components/layout_2020/flow/construct.rs

```diff
@@ -158,25 +162,46 @@ impl BlockContainer {
         }
 
         type Intermediate<Node> = IntermediateBlockLevelBox<Node>;
-        #[derive(Default)]
         struct Target {
             contains_floats: ContainsFloats,
+            outer_content_sizes_of_children: ContentSizes,
+        }
+        impl Default for Target {
+            fn default() -> Self {
+                Self {
+                    contains_floats: ContainsFloats::No,
+                    outer_content_sizes_of_children: ContentSizes::zero(),
+                }
+            }
         }
         let mut target = Target {
             contains_floats: builder.contains_floats,
+            outer_content_sizes_of_children: ContentSizes::zero(),
         };
-        let request_content_sizes = false; // FIXME
+        let request_childrens_outer_content_sizes = request_content_sizes;
```

Comment:
Just pass `request_content_sizes`, no?

Reply:
At some point I had a boolean parameter in `IntermediateBlockLevelBox::finish` (instead of `Option<&mut _>`) and a different name help indicate who requests what from whom, but yeah now it doesn’t matter. I’ll change it.